tests/tests_context.py

```diff
@@ -236,17 +241,19 @@ def get_exception_message(ctx, script):
 
         # todo: trim source line when too long
         with context.Context(self.vm) as ctx:
-            ctx.run_script(script_oops, identifier='my_file_áéíóú.js')
+            ctx.run_script(script_oops, identifier=u'my_file_áéíóú.js'.encode('utf-8'))
```

Comment:
You should not change the tests. If it fails then there is something wrong with the code you are testing. Also, don't add the `u` prefix, import `from __future__ import unicode_literals` instead.